lib/auth/clt.go

```diff
@@ -1482,7 +1469,7 @@ func (c *Client) SetStaticTokens(st types.StaticTokens) error {
 
 // GetLocalClusterName returns local cluster name
 func (c *Client) GetLocalClusterName() (string, error) {
```

Comment:
I can only see [one usage of `GetLocalClusterName()`](https://github.com/gravitational/teleport/blob/f831e446c5b62bf1b1e46b1bb34d4dc4e15a7eca/lib/service/connect.go#L931).

It seems like it might be safe to do away with `GetLocalClusterName()` and replace that usage with `GetDomainName()` ?